### PR TITLE
webpack 5, and HMR webpack style

### PR DESCRIPTION
**Pull request by boutell**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -38,14 +38,6 @@ require('apostrophe')({
         className: 'bp-video-widget'
       }
     },
-    // Manages apostrophe's overall asset pipeline
-    '@apostrophecms/asset': {
-      // When not in production, refresh the page on restart
-      options: {
-        refreshOnRestart: true
```
Comment (boutell) [inline app.js:45]: we use webpack hmr now

File: modules/asset/index.js [hunk 2]
```diff
@@ -1 +1,11 @@
-module.exports = {};
+module.exports = {
+  handlers(self, options) {
+    return {
+      '@apostrophecms/page:beforeSend': {
+        webpack(req) {
+          req.data.isDev = (process.env.NODE_ENV !== 'production');
```
Comment (boutell) [inline modules/asset/index.js:6]: necessary for hmr

File: package.json [hunk 3]
```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "start": "node app",
     "build": "webpack && node app @apostrophecms/asset:build",
-    "dev": "nodemon --delay 1000ms -x \"npm run build && nodemon app.js\"",
-    "storybook": "node app @apostrophecms/storybook:run"
```
Comment (boutell) [inline package.json:10]: extinct

File: .eslintrc [hunk 4]
```diff
@@ -7,6 +7,7 @@
   },
   "rules": {
     "no-var": "error",
-    "no-console": 0
+    "no-console": 0,
+    "multiline-ternary": 0
```
Comment (abea) [inline .eslintrc:11]: I'm wondering if there's some local copy interaction causing your error. The build is running fine when I remove this.

**Review by abea — APPROVED** [review]

**Pull request merged**